tests/nested/manual/core20-fde-dbx/task.yaml

```diff
@@ -141,6 +160,22 @@ execute: |
 
     fetch_and_check_reseal_count_equal "$((reseal_count_start + 7))"
 
+    echo "Update content of DBX"
+    initial_dbx_entries_count="$(cat initial-dbx-entries-count)"
+    # get initial state of all variables
+    remote.exec sudo test-snapd-efitools.tool efi-readvar 2>&1 | tee efi-vars.log
+    remote.push dbx-1-update.auth
+    remote.exec "sudo chattr -i /sys/firmware/efi/efivars/dbx-*"
+    # update dbx
+    remote.exec sudo test-snapd-efitools.tool efi-readvar -v dbx 2>&1 | tee efi-dbx-before.log
+    # expecting 2 entries in the list
+    grep -c 'List ' efi-dbx-before.log | MATCH "$initial_dbx_entries_count"
+    remote.exec sudo test-snapd-efitools.tool efi-updatevar -a -f dbx-1-update.auth dbx
+    remote.exec sudo test-snapd-efitools.tool efi-readvar -v dbx 2>&1 | tee efi-dbx-after.log
+    # expecting 3 entries in the list
+    grep -c 'List ' efi-dbx-after.log | MATCH "$((initial_dbx_entries_count + 1))"
+
+    # reboot
```

Comment:
Should we also test if there is no reboot between "prepare" and "cleanup"? The reboot in between is good test, but I suppose this is not the normal case.

Reply:
I've added this scenario now